scaffold/cmd/templates/plugin-config-ui/README.md

```diff
@@ -0,0 +1,40 @@
+# Getting Started with custom Plugin UI
+
+### `npm install`
+
+This will install all the dependencies required for the Plugin UI to run.
+
+## Available Scripts
+
+In the project directory, you can run:
+
+### `npm start`
+
+Runs the app in the development mode.\
+Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
```

Comment:
I think you mentioned we should run this on port 3001 by default, so that it can work with the cloud UI in development mode?